examples/attention/opencog.conf

```diff
@@ -0,0 +1,137 @@
+#
+# This file provides an example configuration for a production OpenCog
+# server. Particularly noteworthy is the list of automatically loaded
+# modules.
+#
+SERVER_PORT           = 17001
+LOG_FILE              = /tmp/cogserver.log
+
```

Comment:
after skimming the above code, I don't believe that "examples" is really the correct directory for this -- examples are really meant to be tutorials for new users, and this seems to be far beyond a tutorial.

Perhaps this code should go into a directory called `opencog/attention/experiments` ? or something like that?  We certainly have other code, similar to this, that lives in the subdirectory of the subsystem that it is experimenting with.